src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomCS = new Random();

```

Comment:
The variable name `randomCS` could be simplified to `random` for better readability and to follow common naming conventions.